src/task/task.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { Body, Controller, Get, Post, Param, Patch } from "@nestjs/common";
+import { TaskDTO } from "./task.dto";
+
+import { Task } from "./task.entity";
+import { TaskService } from "./task.service";
+
+@Controller("task")
+export class TaskController {
+	public constructor(
+		private taskService: TaskService
+	) { }
+
+	@Post()
+	public addTask(@Body() taskDTO: TaskDTO): void {
+		this.taskService.insertTask(taskDTO);
+	}
+
+	@Get()
+	public getAllTasks(): Promise<Task[]> {
+		return this.taskService.prepareAllTasks();
```

Comment:
```suggestion
		return this.taskService.getAllTasks();
```

it is ok to have the same method names as service